sml_builder/static/js/cookies_function.js

```diff
@@ -0,0 +1,189 @@
+const DEFAULT_COOKIE_CONSENT = {
+    essential: true,
+    settings: false,
+    usage: false,
+    campaigns: false,
+  };
+  
+const COOKIE_CATEGORIES = {
```

Comment:
Where do these entries come from? Did you find some examples in the design system etc?   

I don't think we need all of these:
_ga_D60NB8YQQV: 'usage',
  _ga_F8PLZD05SG: 'usage',
  _ga_T2YY6YFBH2: 'usage',
  _ga_K9VRXZMXMJ: 'usage',
  _ga_WK76MC47: 'usage',
  
Are they the measurements IDs or the container IDs ?

Reply:
Hi yes these are the stream measurement id's and the container id. Its in relation to the cookie info found in the dev tools

Reply:
I will see if we can leave them out :)

Reply:
removed :)

Reply:
I tested on your branch and this seems to work ok, I was expecting we would need to add _ga_MeasurementID but it looks like it is working fine without.